tests/samplers_tests/test_samplers.py

```diff
@@ -41,14 +42,20 @@
         ),
         lambda: optuna.integration.PyCmaSampler(n_startup_trials=0),
         optuna.samplers.NSGAIISampler,
-    ],
+    ]
+    + []
```

Comment:
The syntax for checking to test this if Python >3.6 feels out of place. Didn't see any other tests checking `sys.info_version`. Botorch supports Python 3.6+, so can you let me know why that is necessary?

Reply:
maybe I'm missing something but BoTorch does not support python 3.6.

https://github.com/pytorch/botorch/blob/70143a73fec92d0df9ed6707baedf9215b8ac44d/setup.py#L13-L14

Reply:
Gotcha. Not sure where I saw that. Thanks!